esphome/const.py

```diff
@@ -71,6 +71,7 @@
 CONF_BRIGHTNESS = "brightness"
 CONF_BROKER = "broker"
 CONF_BSSID = "bssid"
+CONF_BTM = "btm"
```

Comment:
For config options we expose to the _user_, we should be more descriptive with names (see for example `enable_mdns` key).
```suggestion
CONF_ENABLE_BTM = "enable_btm"
```